satpy/modifiers/__init__.py

```diff
@@ -19,7 +19,11 @@
 
 # file deepcode ignore W0611: Ignore unused imports in init module
 
-from .base import ModifierBase  # noqa: F401
-from .spectral import NIREmissivePartFromReflectance, NIRReflectance  # noqa: F401
-from .geometry import SunZenithCorrector, EffectiveSolarPathLengthCorrector  # noqa: F401
-from .atmosphere import PSPRayleighReflectance, PSPAtmosphericalCorrection, CO2Corrector  # noqa: F401
+from .base import ModifierBase  # noqa: F401, isort: skip
+from .atmosphere import CO2Corrector  # noqa: F401
+from .atmosphere import PSPAtmosphericalCorrection  # noqa: F401
+from .atmosphere import PSPRayleighReflectance  # noqa: F401
```

Comment:
Same here.  Why make these imports more verbose by repeating the `from .atmosphere import ` part?  Isn't this a violation of DRY?